docs/stencil-docs/getting-started/stencil-technology-stack.md

```diff
@@ -90,7 +90,7 @@ As a developer, you can use Handlebars.js syntax to access objects and use them
 
 Stencil Objects are categorized as either Global, Common, or Other, which is representative of the object's scope or where in the theme it can be accessed. For example, Global Objects are components shared across the entire BigCommerce storefront.
 
-## YAML Front Matter Overview
+## YAML front matter overview
 

```

Comment:
Front matter was capitalized in a different file (Using Front Matter PR #528).  We should be consistent.